cookbooks/bcpc/recipes/graphite.rb

```diff
@@ -113,6 +113,17 @@
   recursive true
 end
 
+directory "#{node['bcpc']['graphite']['local_log_dir']}/webapp" do
+  owner "www-data"
+  group "www-data"
+end
+
+["info.log", "exception.log" ].each do |f|
+  file "#{node['bcpc']['graphite']['local_log_dir']}/webapp/#{f}" do
```

Comment:
On a new cluster, the "webapp" directory and files "info.log" and "exception.log" are owned by root. Unless something touches the files, webapp will never be able to write to those files.